fastn-core/src/migrations.rs

```diff
@@ -0,0 +1,210 @@
+const MIGRATION_TABLE: &str = r#"
+
+CREATE TABLE IF NOT EXISTS
+    fastn_migration
+(
+    id               INTEGER PRIMARY KEY,
+    app_name         TEXT NOT NULL,
+    migration_number INTEGER NOT NULL UNIQUE,
+    migration_name   TEXT NOT NULL,
+    applied_on       INTEGER NOT NULL
+) STRICT;
+
+"#;
+
+pub(crate) async fn migrate(config: &fastn_core::Config) -> Result<(), MigrationError> {
+    let db = config.get_db_url().await;
+    create_migration_table(config, db.as_str()).await?;
+
+    if !has_new_migration(config, db.as_str()).await? {
```

Comment:
Rename to `has_unapplied_migrations()`.

Reply:
Not following this in favor of https://github.com/fastn-stack/fastn/pull/1894#discussion_r1627130340